board/crosscompile-toolchain.cmake

```diff
@@ -0,0 +1,38 @@
+## This file handles cross-compilation configurations for aarch64,
+## known as arm64. The objective of this file is to find and assign
+## cross-compiler and the entire toolchain.
+## It works best with buildroot toolchain, when using it the user
+## needs to set the: TOOLCHAIN_PREFIX and CMAKE_SYSROOT from the
+## command line.
+## Currently, we recommend using buildroot toolchain for
+## cross-compilation. Here is the link to download the toolchains:
+## https://toolchains.bootlin.com/
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSROOT)
+set(TOOLCHAIN_PREFIX "" CACHE STRING "Path for Toolchain for cross compiler and other compilation tools.")
+
+# Without that flag CMake is not able to pass test compilation check.
```

Comment:
```suggestion
# Ensure that CMake tries to build static libraries when testing the compiler.
```
This might be a little more accurate---up to you if you want to change it. :+1: